<?xml version="1.0" encoding="utf-8"?>
<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.2" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 xliff-core-1.2-transitional.xsd">
  <file datatype="xml" original="markdown" source-language="en-US" target-language="it-it">
    <header>
      <tool tool-id="mdxliff" tool-name="mdxliff" tool-version="1.0-192e1fd" tool-company="Microsoft" />
      <xliffext:olfilehash xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">a99f6363a055cbd8272c7514eb695af7843d28b2</xliffext:olfilehash>
      <xliffext:olfilepath xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">fulldocset\System.ContextMarshalException.yml</xliffext:olfilepath>
      <xliffext:oltranslationpriority xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">fulldocset</xliffext:oltranslationpriority>
      <xliffext:oltranslationtype xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">Human Translation</xliffext:oltranslationtype>
      <xliffext:olskeletonhash xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">99c3297e0277268ff38951652df71f61d6bd78ad</xliffext:olskeletonhash>
      <xliffext:olxliffhash xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">24008431dee4e83f1d771ae3553ecbbea209b0c8</xliffext:olxliffhash>
    </header>
    <body>
      <group id="content" extype="content">
        <trans-unit id="101" translate="yes" xml:space="preserve">
          <source>The exception that is thrown when an attempt to marshal an object across a context boundary fails.</source>
          <target state="translated">Eccezione generata quando un tentativo di effettuare il marshalling di un oggetto attraverso un limite di contesto non riesce.</target>       </trans-unit>
        <trans-unit id="102" translate="yes" xml:space="preserve" extradata="MT">
          <source>Objects can marshal by value or by reference.</source>
          <target state="translated">Gli oggetti possono effettuare il marshalling per valore o riferimento.</target>       </trans-unit>
        <trans-unit id="103" translate="yes" xml:space="preserve" extradata="MT">
          <source>Any attempt to pass a type that cannot be marshaled through a context boundary will result in a ContextMarshalException.</source>
          <target state="translated">Qualsiasi tentativo di passare un tipo che non può essere sottoposto a marshalling attraverso un limite di contesto comporterà un ContextMarshalException.</target>       </trans-unit>
        <trans-unit id="104" translate="yes" xml:space="preserve" extradata="MT">
          <source>ContextMarshalException uses the HRESULT COR_E_CONTEXTMARSHAL which has the value 0x80131504L.</source>
          <target state="translated">ContextMarshalException utilizza HRESULT COR_E_CONTEXTMARSHAL, con valore 0x80131504L.</target>       </trans-unit>
        <trans-unit id="105" translate="yes" xml:space="preserve" extradata="MT">
          <source>ContextMarshalException uses the default &lt;xref:System.Object.Equals%2A&gt; implementation, which supports reference equality.</source>
          <target state="translated">ContextMarshalException utilizza il valore predefinito &lt;xref:System.Object.Equals%2A&gt;implementazione, che supporta l'uguaglianza dei riferimenti.&lt;/xref:System.Object.Equals%2A&gt;</target>       </trans-unit>
        <trans-unit id="106" translate="yes" xml:space="preserve" extradata="MT">
          <source>For a list of initial property values for an instance of ContextMarshalException, see the &lt;xref:System.ContextMarshalException.%23ctor%2A&gt; constructors.</source>
          <target state="translated">Per un elenco di valori di proprietà iniziali per un'istanza di ContextMarshalException, vedere il &lt;xref:System.ContextMarshalException.%23ctor%2A&gt;costruttori.&lt;/xref:System.ContextMarshalException.%23ctor%2A&gt;</target>       </trans-unit>
        <trans-unit id="107" translate="yes" xml:space="preserve">
          <source>Initializes a new instance of the <bpt id="p1">&lt;xref href="System.ContextMarshalException"&gt;</bpt><ept id="p1">&lt;/xref&gt;</ept> class with default properties.</source>
          <target state="translated">Inizializza una nuova istanza di <bpt id="p1">&lt;xref href="System.ContextMarshalException"&gt;</bpt> <ept id="p1">&lt;/xref&gt;</ept> classe con le proprietà predefinite.</target>       </trans-unit>
        <trans-unit id="108" translate="yes" xml:space="preserve" extradata="MT">
          <source>The following table shows the initial property values for an instance of &lt;xref:System.ContextMarshalException&gt;.</source>
          <target state="translated">Nella tabella seguente vengono illustrati i valori di proprietà iniziali per un'istanza di &lt;xref:System.ContextMarshalException&gt;.&lt;/xref:System.ContextMarshalException&gt;</target>       </trans-unit>
        <trans-unit id="109" translate="yes" xml:space="preserve" extradata="MT">
          <source>|Property|Value|   |--------------|-----------|   |&lt;xref:System.Exception.InnerException%2A&gt;|<ph id="ph1">`null`</ph>.|   |&lt;xref:System.Exception.Message%2A&gt;|The empty string ("").|</source>
          <target state="translated">| Proprietà | Valore |   |--------------|-----------|   | &lt;xref:System.Exception.InnerException%2A&gt;| <ph id="ph1">`null`</ph>.|   | &lt;xref:System.Exception.Message%2A&gt;| Una stringa vuota (""). |&lt;/xref:System.Exception.Message%2A&gt; &lt;/xref:System.Exception.InnerException%2A&gt;</target>       </trans-unit>
        <trans-unit id="110" translate="yes" xml:space="preserve">
          <source>Initializes a new instance of the <bpt id="p1">&lt;xref href="System.ContextMarshalException"&gt;</bpt><ept id="p1">&lt;/xref&gt;</ept> class with a specified error message.</source>
          <target state="translated">Inizializza una nuova istanza di <bpt id="p1">&lt;xref href="System.ContextMarshalException"&gt;</bpt> <ept id="p1">&lt;/xref&gt;</ept> classe con un messaggio di errore specificato.</target>       </trans-unit>
        <trans-unit id="111" translate="yes" xml:space="preserve" extradata="MT">
          <source>The following table shows the initial property values for an instance of &lt;xref:System.ContextMarshalException&gt;.</source>
          <target state="translated">Nella tabella seguente vengono illustrati i valori di proprietà iniziali per un'istanza di &lt;xref:System.ContextMarshalException&gt;.&lt;/xref:System.ContextMarshalException&gt;</target>       </trans-unit>
        <trans-unit id="112" translate="yes" xml:space="preserve" extradata="MT">
          <source>|Property|Value|   |--------------|-----------|   |&lt;xref:System.Exception.InnerException%2A&gt;|<ph id="ph1">`null`</ph>.|   |&lt;xref:System.Exception.Message%2A&gt;|The error message string.|</source>
          <target state="translated">| Proprietà | Valore |   |--------------|-----------|   | &lt;xref:System.Exception.InnerException%2A&gt;| <ph id="ph1">`null`</ph>.|   | &lt;xref:System.Exception.Message%2A&gt;| La stringa di messaggio di errore. |&lt;/xref:System.Exception.Message%2A&gt; &lt;/xref:System.Exception.InnerException%2A&gt;</target>       </trans-unit>
        <trans-unit id="113" translate="yes" xml:space="preserve">
          <source>The error message that explains the reason for the exception.</source>
          <target state="translated">Il messaggio di errore che spiega il motivo dell'eccezione.</target>       </trans-unit>
        <trans-unit id="114" translate="yes" xml:space="preserve">
          <source>Initializes a new instance of the <bpt id="p1">&lt;xref href="System.ContextMarshalException"&gt;</bpt><ept id="p1">&lt;/xref&gt;</ept> class with serialized data.</source>
          <target state="translated">Inizializza una nuova istanza di <bpt id="p1">&lt;xref href="System.ContextMarshalException"&gt;</bpt> <ept id="p1">&lt;/xref&gt;</ept> classe con dati serializzati.</target>       </trans-unit>
        <trans-unit id="115" translate="yes" xml:space="preserve">
          <source>The <bpt id="p1">&lt;xref href="System.Runtime.Serialization.SerializationInfo"&gt;</bpt><ept id="p1">&lt;/xref&gt;</ept> that holds the serialized object data about the exception being thrown.</source>
          <target state="translated">Il <bpt id="p1">&lt;xref href="System.Runtime.Serialization.SerializationInfo"&gt;</bpt> <ept id="p1">&lt;/xref&gt;</ept> che contiene i dati oggetto serializzati relativi all'eccezione generata.</target>       </trans-unit>
        <trans-unit id="116" translate="yes" xml:space="preserve">
          <source>The &lt;xref:System.Runtime.Serialization.StreamingContext&gt; that contains contextual information about the source or destination.</source>
          <target state="translated">Il &lt;xref:System.Runtime.Serialization.StreamingContext&gt;che contiene le informazioni contestuali sull'origine o destinazione.&lt;/xref:System.Runtime.Serialization.StreamingContext&gt;</target>       </trans-unit>
        <trans-unit id="117" translate="yes" xml:space="preserve">
          <source>Initializes a new instance of the <bpt id="p1">&lt;xref href="System.ContextMarshalException"&gt;</bpt><ept id="p1">&lt;/xref&gt;</ept> class with a specified error message and a reference to the inner exception that is the cause of this exception.</source>
          <target state="translated">Inizializza una nuova istanza di <bpt id="p1">&lt;xref href="System.ContextMarshalException"&gt;</bpt> <ept id="p1">&lt;/xref&gt;</ept> classe con un messaggio di errore specificato e un riferimento all'eccezione interna che è la causa dell'eccezione corrente.</target>       </trans-unit>
        <trans-unit id="118" translate="yes" xml:space="preserve" extradata="MT">
          <source>An exception that is thrown as a direct result of a previous exception should include a reference to the previous exception in the &lt;xref:System.Exception.InnerException%2A&gt; property.</source>
          <target state="translated">Un'eccezione generata come risultato diretto di un'eccezione precedente deve includere un riferimento all'eccezione precedente nella &lt;xref:System.Exception.InnerException%2A&gt;proprietà.&lt;/xref:System.Exception.InnerException%2A&gt;</target>       </trans-unit>
        <trans-unit id="119" translate="yes" xml:space="preserve" extradata="MT">
          <source>The &lt;xref:System.Exception.InnerException%2A&gt; property returns the same value that is passed into the constructor, or <ph id="ph1">`null`</ph> if the &lt;xref:System.Exception.InnerException%2A&gt; property does not supply the inner exception value to the constructor.</source>
          <target state="translated">Il &lt;xref:System.Exception.InnerException%2A&gt;proprietà restituisce lo stesso valore passato al costruttore o <ph id="ph1">`null`</ph> se il &lt;xref:System.Exception.InnerException%2A&gt;proprietà non fornisce il valore di eccezione interna al costruttore.&lt;/xref:System.Exception.InnerException%2A&gt; &lt;/xref:System.Exception.InnerException%2A&gt;</target>       </trans-unit>
        <trans-unit id="120" translate="yes" xml:space="preserve" extradata="MT">
          <source>The following table shows the initial property values for an instance of &lt;xref:System.ContextMarshalException&gt;.</source>
          <target state="translated">Nella tabella seguente vengono illustrati i valori di proprietà iniziali per un'istanza di &lt;xref:System.ContextMarshalException&gt;.&lt;/xref:System.ContextMarshalException&gt;</target>       </trans-unit>
        <trans-unit id="121" translate="yes" xml:space="preserve" extradata="MT">
          <source>|Property|Value|   |--------------|-----------|   |&lt;xref:System.Exception.InnerException%2A&gt;|The inner exception reference.|   |&lt;xref:System.Exception.Message%2A&gt;|The error message string.|</source>
          <target state="translated">| Proprietà | Valore |   |--------------|-----------|   | &lt;xref:System.Exception.InnerException%2A&gt;| Il riferimento all'eccezione interna. |   | &lt;xref:System.Exception.Message%2A&gt;| La stringa di messaggio di errore. |&lt;/xref:System.Exception.Message%2A&gt; &lt;/xref:System.Exception.InnerException%2A&gt;</target>       </trans-unit>
        <trans-unit id="122" translate="yes" xml:space="preserve">
          <source>The error message that explains the reason for the exception.</source>
          <target state="translated">Il messaggio di errore che spiega il motivo dell'eccezione.</target>       </trans-unit>
        <trans-unit id="123" translate="yes" xml:space="preserve">
          <source>The exception that is the cause of the current exception.</source>
          <target state="translated">Eccezione che è la causa dell'eccezione corrente.</target>       </trans-unit>
        <trans-unit id="124" translate="yes" xml:space="preserve">
          <source>If the <ph id="ph1">`inner`</ph> parameter is not <bpt id="p1">&lt;xref uid="langword_csharp_null" name="null" href=""&gt;</bpt><ept id="p1">&lt;/xref&gt;</ept>, the current exception is raised in a <bpt id="p2">&lt;xref uid="langword_csharp_catch" name="catch" href=""&gt;</bpt><ept id="p2">&lt;/xref&gt;</ept> block that handles the inner exception.</source>
          <target state="translated">Se la <ph id="ph1">`inner`</ph> parametro non è <bpt id="p1">&lt;xref uid="langword_csharp_null" name="null" href=""&gt;</bpt> <ept id="p1">&lt;/xref&gt;</ept>, l'eccezione corrente viene generata in un <bpt id="p2">&lt;xref uid="langword_csharp_catch" name="catch" href=""&gt;</bpt> <ept id="p2">&lt;/xref&gt;</ept> blocco che gestisce l'eccezione interna.</target>       </trans-unit>
      </group>
    </body>
  </file>
</xliff>